### PR TITLE
Fix awkward sentence in signal docs

### PR DESCRIPTION
**Pull request by strugee**
The diff is kind of hard to read because I reformatted where the paragraph breaks, but all I did was move the position of `also` in the sentence.

### CODE REVIEW

File: Doc/library/signal.rst [hunk 1]
```diff
@@ -711,10 +711,11 @@ case, wrap your entry point to catch this exception as follows::
     if __name__ == '__main__':
         main()
 
-Do not set :const:`SIGPIPE`'s disposition to :const:`SIG_DFL`
-in order to avoid :exc:`BrokenPipeError`.  Doing that would cause
-your program to exit unexpectedly also whenever any socket connection
-is interrupted while your program is still writing to it.
+Do not set :const:`SIGPIPE`'s disposition to :const:`SIG_DFL` in
+order to avoid :exc:`BrokenPipeError`.  Doing that would also
+cause your program to exit unexpectedly whenever any socket
```
Comment (JelleZijlstra) [inline Doc/library/signal.rst]:
```suggestion
order to avoid :exc:`BrokenPipeError`.  Doing that causes
your program to exit unexpectedly whenever any socket
```
More concise wording

Comment (strugee) [reply Doc/library/signal.rst]: Ack, sorry for the delayed response. This doesn't quite feel right since I feel like this should be in the future tense? Maybe just remove "also", though I don't care _that_ much and don't want to kill a bunch of time bikeshedding.

Comment (JelleZijlstra) [reply Doc/library/signal.rst]: Sure, let's change it back to "would cause" and then I'll merge it

File: Doc/library/signal.rst [hunk 2]
```diff
@@ -711,10 +711,11 @@ case, wrap your entry point to catch this exception as follows::
     if __name__ == '__main__':
         main()
 
-Do not set :const:`SIGPIPE`'s disposition to :const:`SIG_DFL`
-in order to avoid :exc:`BrokenPipeError`.  Doing that would cause
-your program to exit unexpectedly also whenever any socket connection
-is interrupted while your program is still writing to it.
+Do not set :const:`SIGPIPE`'s disposition to :const:`SIG_DFL` in
+order to avoid :exc:`BrokenPipeError`.  Doing that causes
```
Comment (JelleZijlstra) [inline Doc/library/signal.rst]:
```suggestion
order to avoid :exc:`BrokenPipeError`.  Doing that would cause
```

**Review by JelleZijlstra — APPROVED** [review]
